lib/Service/ExAppService.php

```diff
@@ -84,6 +84,9 @@ public function registerExApp(array $appInfo): ?ExApp {
 			$this->exAppMapper->insert($exApp);
 			$exApp = $this->exAppMapper->findByAppId($appInfo['id']);
 			$this->cache->remove('/ex_apps');
+			if (isset($appInfo['external-app']['routes'])) {
+				$exApp->setRoutes($this->registerExAppRoutes($exApp, $appInfo['external-app']['routes'])->getRoutes() ?? []);
```

Comment:
can we make that `->getRoutes` will always return `[]`

This will allow is to remove the check with `??` here, right?

Reply:
No, otherwise the DB mapper will try to update this field, which is from another table.